TLSharp.Core/Network/TcpTransport.cs

```diff
@@ -3,43 +3,64 @@
 using System.Net.Sockets;
 using System.Threading;
 using System.Threading.Tasks;
+using Starksoft.Net.Proxy;
```

Comment:
could you please provide the URL which Starksoft obtained of it?
```suggestion
//url: 
using Starksoft.Net.Proxy;
```